doc/usage.md

```diff
@@ -0,0 +1,422 @@
+# lib-generis-search
+
+## service basic usage :
+
+### criteria basic search :
+
+search every items with label equal to 'foo'.
+
+```php
+$search = $this->getServiceManager()->get(\oat\oatbox\search\ComplexSearchService::SERVICE_ID);
```

Comment:
I think this example is showing how to use the complex search in Generis, not as a standalone API right? Are you sure the \oat\oatbox\search\ComplexSearchService is the right namespace :) ?